gateway-ha/src/main/java/io/trino/gateway/ha/router/TrinoQueryProperties.java

```diff
@@ -421,8 +425,7 @@ private QualifiedName qualifyName(QualifiedName name)
     {
         List<String> nameParts = name.getParts();
         return switch (nameParts.size()) {
```

Comment:
There are no break statements in this `switch` clause, is that intentional. What does nameParts contain?